packages/expo-blur/src/__tests__/BlurView-test.web.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { StyleSheet } from 'react-native';
+
+import { BlurView } from '..';
+
+const getStyleProp = (component, prop) => StyleSheet.flatten(component.prop('style'))[prop];
+
+// @ts-ignore
+const originalCSS = global.CSS;
+
+beforeAll(() => {
+  // @ts-ignore
+  global.CSS = {
+    supports() {
+      return true;
+    },
+  };
+});
+
+afterAll(() => {
+  // @ts-ignore
+  global.CSS = originalCSS;
+});
+
+it(`uses a transparent background color when filters aren't supported`, () => {
```

Comment:
I would write two tests here: one for using the backdrop filter when it is supported, and one when it isn't supported. (basically the same test code you have here, just split into two tests)